compiler/src/iree/compiler/Codegen/LLVMGPU/test/linalg_transform.mlir

```diff
@@ -1,9 +1,11 @@
 // RUN: iree-opt %s  --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-lower-executable-target)))" \
-// RUN: --iree-codegen-llvmgpu-use-transform-dialect=%p/transform_dialect_codegen_bufferize_spec.mlir | \
+// RUN:     --iree-codegen-llvmgpu-enable-transform-dialect-jit=false \
+// RUN:     --iree-codegen-llvmgpu-use-transform-dialect=%p/transform_dialect_codegen_bufferize_spec.mlir | \
 // RUN: FileCheck %s
 
 // RUN: iree-opt %s  --pass-pipeline="builtin.module(hal.executable(hal.executable.variant(iree-llvmgpu-lower-executable-target)))" \
-// RUN: --iree-codegen-llvmgpu-use-transform-dialect=%p/transform_dialect_codegen_foreach_to_gpu_spec.mlir | \
+// RUN:     --iree-codegen-llvmgpu-enable-transform-dialect-jit=false \
```

Comment:
THe flags seem strange here....

Reply:
There are two mechanism to use transform dialect from flag that can't be used at the same time. So this needs to disable transform dialect jit. We will clean up the file based mechanism soon based if it is not needed anymore.